src/subwindows/serialSettingsDialog.cpp

```diff
@@ -242,17 +248,17 @@ void SerialSettingsDialog::createForm() {
 
     QHBoxLayout *buttonsLayout = new QHBoxLayout();
 
-    connectButton = new QPushButton(tr("Connect"));
-    disconnectButton = new QPushButton(tr("Disconnect"));
+//    connectButton = new QPushButton(tr("Connect")); // moved to camera settings windows for clearer UX
```

Comment:
If these are moved to another class, just delete them instead of commenting them out.

Reply:
Done